messages.go

```diff
@@ -40,6 +40,10 @@ type ClientInitKey struct {
 	privateKey *HPKEPrivateKey `tls:"omit"`
 }
 
+type ClientInitKeySecrets struct {
+	PrivateKey []byte `tls:"head=2"`
+}
```

Comment:
Rather than this approach, I would prefer we just did something like:

```
func (cik ClientInitKey) GetPrivateKey() (HPKEPrivateKey, bool) { … }
func (cik *ClientInitKey) SetPrivateKey(key HPKEPrivateKey) { /* verify it matches the public key, then store */ }
```